### PR TITLE
feat: Add docs for config options

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

File: docs/dev-guide/configuration.md [hunk 1]
```diff
@@ -0,0 +1,773 @@
+---
+id: dev-guide-configuration
+title: Configuration
+---
+
+:::note NOTE
+Options marked with ***** are not overwritable through `configOverwrite`
```
Comment (saghul) [inline docs/dev-guide/configuration.md:7]: Do we have any of these yet? Can we use something mroe visual perhaps? Like 🚫 or ❌

File: docs/dev-guide/configuration.md [hunk 2]
```diff
@@ -0,0 +1,773 @@
+---
+id: dev-guide-configuration
+title: Configuration
+---
+
+:::note NOTE
```
Comment (saghul) [inline docs/dev-guide/configuration.md:6]: You can drop the "NOTE" the admonition provides a default title IIRC.

File: docs/dev-guide/iframe-commands.md [hunk 3]
```diff
@@ -0,0 +1,584 @@
...
+
+```javascript
+api.executeCommand('muteEveryone', 'video');
+```
+
+### startRecording
+
+Starts a file recording or streaming session using passed parameters:
+
+  - **RTMP streaming** - Recording mode set to **`stream`** with an **`rtmpStreamKey`**. The **`rtmpBroadcastID`** value is optional.
```
Comment (saghul) [inline docs/dev-guide/iframe-commands.md:281]: Note to self, check this, I believee one of the 2 values is completely ignored.

File: docs/dev-guide/iframe.md [hunk 4]
```diff
@@ -188,6 +188,11 @@ const api = new JitsiMeetExternalAPI(domain, options);
 ```
 **Note:** **`TILE_VIEW_MAX_COLUMNS`** accepts values from 1 to 5. The default value is 5.
 
+
+:::warning
```
Comment (saghul) [inline docs/dev-guide/iframe.md:192]: Shouldn't we have this at the top of the file?

**Review by saghul — COMMENTED** [review]
This looks amazing, thanks Robert! Left some minor comments!

File: docs/dev-guide/configuration.md (hunk 1 above)
Comment (robertpin) [reply docs/dev-guide/configuration.md:7]: We do. Will update to use 🚫

File: docs/dev-guide/iframe.md (hunk 4 above)
Comment (robertpin) [reply docs/dev-guide/iframe.md:192]: Well, I don't know about that. The thing is that this doesn't apply to the sections above where it is now

Comment (saghul) [reply docs/dev-guide/iframe.md:192]: Ah right. Let's leave it there then!

**Review by saghul — APPROVED** [review]
🚀

**Pull request merged**